### PR TITLE
added support for allow_disk_use

### PR DESCRIPTION
**Pull request by adamsqi**
Added support for missing allow_disk_use param: https://pymongo.readthedocs.io/en/stable/api/pymongo/collection.html#pymongo.collection.Collection.aggregate

### CODE REVIEW

File: mongomock/collection.py [hunk 1]
```diff
@@ -1018,7 +1018,7 @@ def find(self, filter=None, projection=None, skip=0, limit=0,
              no_cursor_timeout=False, cursor_type=None, sort=None,
              allow_partial_results=False, oplog_replay=False, modifiers=None,
              batch_size=0, manipulate=True, collation=None, session=None,
-             max_time_ms=None, **kwargs):
+             max_time_ms=None, allow_disk_use=False, **kwargs):
```
Comment (pcorpet) [inline mongomock/collection.py:1021]: I believe the default in pymongo (at least in the doc) is None.

**Review by pcorpet — CHANGES_REQUESTED** [review]
Thanks for your change.

Please add a usage in any of existing test or copy an existing one to set it with True or False and check that there are no errors (for regression).

Disregard the broken pylint, I'll take care of it.

File: mongomock/collection.py (hunk 1 above)
Comment (adamsqi) [reply mongomock/collection.py:1021]: Here in the doc: https://pymongo.readthedocs.io/en/stable/api/pymongo/collection.html#pymongo.collection.Collection.aggregate
I can see:
`The default is False`

**Comment by adamsqi** [comment]
@pcorpet 
Hi, I can see there is an issue with:
`W: GPG error: https://packages.erlang-solutions.com/ubuntu focal Release: The following signatures were invalid: BADSIG D208507CA14F4FCA Erlang Solutions Ltd. <packages@erlang-solutions.com>`
in Travis Cl

**Comment by adamsqi** [comment]
Hi @pcorpet, have you had time to look at this PR?

File: mongomock/collection.py (hunk 1 above)
Comment (pcorpet) [reply mongomock/collection.py:1021]: My bad.

**Comment by pcorpet** [comment]
> @pcorpet Hi, I can see there is an issue with: `W: GPG error: https://packages.erlang-solutions.com/ubuntu focal Release: The following signatures were invalid: BADSIG D208507CA14F4FCA Erlang Solutions Ltd. <packages@erlang-solutions.com>` in Travis Cl

Can you try rebasing on HEAD of develop please?

**Comment by pcorpet** [comment]
> Please add a usage in any of existing test or copy an existing one to set it with True or False and check that there are no errors (for regression).

Add a usage of the keyword argument as well in a test.

**Comment by pcorpet** [comment]
Thanks for your changes!

**Comment by adamsqi** [comment]
@pcorpet thank you! When do you plan to release a new version?

**Pull request merged**